maintenance/maintenance.html

```diff
@@ -114,37 +114,34 @@ <h4 class="h3-heading mb-4">Let’s protect the world’s largest resource for f
     <footer class="site-footer dark-theme py-5">
       <div class="container">
         <div class="row">
-          <div class="col-md-5">
-            <div class="my-5">
-              <a class="logo" href="https://mozilla.org"><img src="static/images/mozilla-block-white.svg"></a>
-            </div>
+          <div class="col-md-5 col-lg-6 d-flex flex-column justify-content-between">
+            <a class="logo" href="https://mozilla.org"><img src="static/images/mozilla-block-white.svg" width="96"></a>
           </div>
-          <div class="col-md-6 offset-md-1">
+          <div class="col-md-6 col-lg-5 offset-md-1 d-flex flex-column justify-content-between">
             <div class="row">
-              <div class="col-lg-4">
+              <div class="col-lg-6 col-xl-5">
                 <h5 class="h5-heading">More about us</h5>
                 <ul class="d-flex">
                   <li class="mr-2"><a class="twitter-glyph small d-inline-block" href="https://twitter.com/mozilla"><span class="sr-only">Twitter</span></a></li>
                   <li class="mr-2"><a class="instagram-glyph small d-inline-block" href="https://www.instagram.com/mozilla/"><span class="sr-only">Instagram</span></a></li>
                   <li class="mr-2"><a class="github-glyph small d-inline-block" href="https://www.github.com/mozilla/foundation.mozilla.org/"><span class="sr-only">Github</span></a></li>
                 </ul>
               </div>
-              <div class="col-lg-8">
+              <div class="col-lg-6 col-xl-7">
                 <ul class="link-list list-unstyled mb-0">
                   <li class="mb-2"><a id="donate-footer-btn" href="https://donate.mozilla.org/" target="_blank" rel="noopener noreferrer" class="dark-theme">Donate</a></li>
+                  <li class="mb-2"><a href="https://www.mozilla.org/privacy/websites/#cookies" class="dark-theme">Cookies</a></li>
                   <li class="mb-2"><a href="https://www.mozilla.org/about/legal/terms/mozilla/" class="dark-theme">Legal</a></li>
-                  <li class="mb-2"><a href="https://creativecommons.org/licenses/by/4.0" class="dark-theme">License</a></li>
                   <li class="mb-2"><a href="https://www.mozilla.org/about/governance/policies/participation/" class="dark-theme">Participation Guidelines</a></li>
+                  <li><a href="https://foundation.mozilla.org/en/press-center/" class="dark-theme">Press Center</a></li>

```

Comment:
does this `<li>` not need a `class="mb-2"`?

Reply:
It should. Good catch!